src/App.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import { hot } from 'react-hot-loader'
+// import DataGlobe from 'components/globe-data-view';
+// import dataGlobeViewConfig from './sceneConfigs/dataGlobeViewConfig';
+import featuredGlobeViewConfig from './sceneConfigs/featuredGlobeViewConfig';
+import FeaturedGlobe from 'components/globe-featured-view';
+import Sidebar from 'components/shared/sidebar';
+
+class App extends Component {
+  render() {
+    return (
+      <div className="App">
+        <FeaturedGlobe sceneConfig={featuredGlobeViewConfig} />
+        <Sidebar>
```

Comment:
unless... we could also pull the contents of the sidebar from redux. For example, we could have a list of all widgets in redux with info if they are currently activated or not, that way we could have it updated without checking it on the Sidebar level. I.e:
```
<Sidebar>
  <SearchWidget />
  <PrioritasionWidget />
  ...
  <SomeWidgetFromFeaturedGlobe />
</Sidebar>
```
with this approach the specific widgets would have the logic if they're displayed or not.
Although, now that I wrote this I realized that we can't really make it this way, as the order of the widgets will also be changing. So I think the best way to handle that would be to have two paths with two sidebars.

Reply:
Yeah definitely having two different 'paths' for each 'mode' of the app will be the best way forward.
We may be sharing some bits like `router`, `redux-modules`, `selectors`... but yeah I think this would be the right approach.
Thank you!!